web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/CSIntegrationService.java

```
@@ -681,4 +683,16 @@ public Question sendAnswer(SendAnswerRequestDTO request, String messageId) {
 
         return response.getQuestion();
     }
+
+    public Certificate forwardCertificate(String certificateId, ForwardCertificateRequestDTO request) {
+        final var url = baseUrl + CERTIFICATE_ENDPOINT_URL + "/" + certificateId + "/forward";
+
+        final var response = restTemplate.postForObject(url, request, CertificateServiceForwardCertificateResponseDTO.class);
+
+        if (response == null) {
```

Throw an exception instead, so that we fail fast!